hazelcast/src/test/java/com/hazelcast/query/impl/getters/GetterFactoryTest.java

```diff
@@ -58,6 +77,15 @@ public void newFieldGetter_whenExtractingFromSimpleField_thenInferTypeFromFieldT
     }
 
     @Test
+    public void newMethodGetter_whenExtractingFromSimpleField_thenInferTypeFromFieldType() throws Exception {
```

Comment:
Thx for your comment. IMHO long names are often the only way to express the intent of this test.
I think we can live with this ones too.